### DISCUSSION TITLE
How to draw perfect horizontal or vertical lines with FREEHAND tool (H)?

### DISCUSSION THREAD

**Discussion by iberck** [pubpub-zz/ppInk#221] [Q&A] [answered]
I am aware that using the line tool and pressing shift draws perfect horizontal/vertical lines.

Are there any way to draw perfect horizontal or vertical lines with freehand tool (H) ?

**Comment by pubpub-zz** [comment]
What do you mean : if shift is pressed, the freehand tool would behave like the line tool with shift pressed ?
see #223

**Reply by iberck** [reply]
Yes, tools like demohelper allow you to draw horizontal/vertical lines while you press shift, and draw straight lines while you press ctrl. This behavior allows you to draw freehand and at the same time draw lines very quickly and is very useful.

I was wondering if there is currently something to have similar behavior within pplnk?

it would be awesome to have!

**Comment by pubpub-zz** [comment, marked as answer]
see #226
